conf/test_tools_somatic_ascat.config

```diff
@@ -0,0 +1,26 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Nextflow config file for running minimal tests
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Defines input files and everything required to run a fast and simple pipeline test.
+
+    Use as follows:
+        nextflow run nf-core/sarek -profile test,<extra_test_profile>,<docker/singularity> --outdir <OUTDIR>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+*/
+
+params {
+    input               = "${baseDir}/tests/csv/3.0/ascat_somatic.csv"
+    genome              = 'GATK.GRCh37'
+    igenomes_ignore     = false
+    ascat_chromosomes   = 'c("21", "22")'
+    ascat_loci          = "/mnt/volume/repos/modules/test_ascat2/G1000_loci_hg19.zip"
```

Comment:
Hard-coded path?

Reply:
Yes, something that will be changed in a separate PR, this are manual tests